### PR TITLE
Update to rails 7 (and other gems)

### PR DESCRIPTION
**Pull request by dmitryrck**
I am going to test on my own instance and then report back, but so far that is pretty much it :).

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 source 'https://rubygems.org'
 
-gem "rails", "6.0.4.7"
+ruby "2.7.5"
```
Comment (dmitryrck) [inline Gemfile:3]: I don't like forcing ruby version, but without this Heroku does not install the correct one :/

Comment (rafaelrpbelo) [reply Gemfile:3]: What about ruby 3?

**Review by rafaelrpbelo — APPROVED** [review]
LGTM! :tada: 

lgtm

File: Gemfile (hunk 1 above)
Comment (rafaelrpbelo) [reply Gemfile:3]: You could also specify ruby version through an [env var](https://media.giphy.com/media/3o72FcJmLzIdYJdmDe/giphy.gif).
I don't like it though.

Comment (dmitryrck) [reply Gemfile:3]: > What about ruby 3?

I wish, I can manage to install on my linux laptop. And it works fine inside docker as well.

But I don't know why I cannot manage to compile any ruby version on the company's macbook :\.

So, I will leave it for a while. In the future ruby 3 is going to be everywhere. The same way it happened with ruby 2 that replaced 1.9.

**Pull request merged**